src/serdes/binary-parser.ts

```diff
@@ -107,10 +107,26 @@ class BinaryParser {
    * @return Field ordinal
    */
   readFieldOrdinal(): number {
-    const tagByte = this.readUInt8();
-    const type = (tagByte & 0xf0) >>> 4 || this.readUInt8();
-    const nth = tagByte & 0x0f || this.readUInt8();
-    return (type << 16) | nth;
+    let type = this.readUInt8();
+    let field = type & 15;
```

Comment:
in the rippled code, this is called `name`, but here we are calling it `field`. is there a reason for the difference? which name is better?

Reply:
I called it field to be consistent with the Docs, could also call it `name` or `nth`

Reply:
We call this `nth` elsewhere right? Would prefer we be consistent within this codebase